docs/src/components/productX/XComponents.tsx

```diff
@@ -93,7 +93,7 @@ export default function XComponents() {
                 )}
               </Highlighter>
             ))}
-            <More href={ROUTES.roadmap} />
+            <More href={ROUTES.xDocs} />
```

Comment:
Discussed with José
The roadmap is totally out of date so it's better to link to the root of our doc.
We can probably have a better page in the future but the roadmap is just bad right now.

Reply:
Makes a lot of sense IMO 👍